src/test/resources/pages/instructorFeedbackAllSessionTypes.html

```diff
@@ -758,6 +758,8 @@ <h5 class="col-xs-2" data-original-title="Select this option to use a custom tim
       <input name="user" type="hidden" value="CFeedbackUiT.instructor">
       <input name="token" type="hidden" value="${sessionToken}">
     </form>
+    <div data-value="" name="editsessiondetails">
```

Comment:
@whipermr5 Since the only other place it is used is the `instructorFeedbackSessionsPage`, where it is in edit mode by default (and indeed should be), I think passing `false` would be misleading. I also don't think it is necessary to pass the variable down there at all. Should I instead put a test and add in the data attribute only if the variable exists?